test/test.js

```diff
@@ -7,3 +18,9 @@ import "../src/import-export-steal-test";
 import "../src/mixin-bindings-test";
 import "../src/mixin-initialize-bindings-test";
 import "../src/mixin-bind-behaviour-test";
+
+if (supportsClassFields) {
+	//It doesn't work with require
+	//Even when change the above imports to require
+	steal.import('~/src/mixin-props-class-fields-test');
```

Comment:
I just tried this out in the canjs/canjs suite and this file can't be loaded since the `~` is the `canjs` folder. We'll need to figure out how to do this.